src/components/RocketsView.js

```diff
@@ -1,13 +1,20 @@
-import React, { useState } from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Container from 'react-bootstrap/Container';
 import RocketList from './RocketList';
-import { ships } from '../data/mock';
+import { fetchRocketsThunk } from '../redux/slices/rocketsSlice';
 
 const RocketsView = () => {
-  const [rockets] = useState(ships);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRocketsThunk());
```

Comment:
That works fine, but there's a note on the ticket #17, it says:

> NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation).

So I think we should dispatch the fetch actions in a wrapper component, for me I dispatch in the `Layout` component.